packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -98,6 +99,7 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         currency,
         fetchedToken,
         OCAPISessionsURL,
+        enablePWAKitPrivateClient,
```

Comment:
How about updating the jsdoc comment (on lines above 87) for this `CommerceApiProvider` component? It'd be useful for those developers who do not read the online documentation about private client.

Reply:
I've updated the comment to add more details about `enablePWAKitPrivateClient` and `clientSecret`

Reply:
Excellent, thank you!